### DISCUSSION TITLE
Küplerden oluşan piramit sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#178] [Soru Cevap] [answered]
![b31i4r5o0729](https://github.com/sonsuzus/sonsuzus/assets/52765796/0003a1f0-23a3-433b-99ce-8c2486713ea5)

Küplerden oluşan bir piramitin birinci katında 1, ikinci katında 3, üçüncü katında 6 küp bulunmaktadır.
Şayet bu piramit 100 katlı olsaydı toplam kaç küp bulunacaktır?

**Comment by Berkek007** [comment, marked as answer]
```
kupsayisi = 1
kat = 1
art = 3

while kat <= 99:

    kat += 1

    kupsayisi = kupsayisi + art

    art = kat + 1 + art


    

print(kat," katlı bir piramitte ",kupsayisi," küp bulunur. ")
```


Burada örüntü 1, 4, 10, 20, 35 .... şeklinde gider.
art adını verdiğim değişken ise buradaki artışı kata bağlı olarak gösteriyor.
